### ISSUE TITLE
Install/implementation fail

### ISSUE THREAD

**Issue by benehudson** [ArnoldV/Our.Umbraco.GMaps#89]
Yesterday I installed Gmaps via nuget.  Now I'm attempting to run Gmaps 2.0.5 on an Umbraco 8.17.2 site.  I'm getting an error in the backoffice when I attempt to edit content on a page whose doctype contains a Gmaps property editor.

The error message is:
System.InvalidOperationException: Type 'Our.Umbraco.GMaps.ProprtyValueConverter.GMapsPropertyValueConverter' cannot be an IPropertyValueConverter for property 'location' of content type 'community' because type 'Our.Umbraco.GMaps.PropertyValueConverter.SingleMapPropertyValueConverter' has already been detected as a converter for that property, and only one converter can exist for a property.

Community is the doctype.  Location is the property whose editor is Gmaps.

I've also posted in the Umbraco forums, but no replies...
https://our.umbraco.com/forum/using-umbraco-and-getting-started//108223-ourumbracogmaps-install-fail

Wondering if I need to revert to a previous version of Gmaps, or how you think might be best to resolve this issue.

**Comment by robertjf** [comment]
Hi @benehudson is this a fresh install or an upgrade?  Make sure that you don't have a stale dll, the current version does not contain a `GMapsPropertyValueConverter` class, it was renamed to `SingleMapPropertyValueConverter` for future compatibility.

**Comment by benehudson** [comment]
@robertjf this was a fresh install.  When I did it, I didn't see the property editor in the back-office.  Looking back I think b/c the App_Plugin folder was excluded from my project.  So, I tried installing via the packages tab, which I think gave me Gmaps 1.2, then I could see the prop editor, but that's when the error started.  So I uninstalled both via the packages tab, and via nuget, and started over with nuget.  That's where I am now.  Packages.config says I'm running 2.0.5  Installed.Packages is blank...should that be?
